<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Modder ver 0.1 by ideal.exe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="listLayout">
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>登録データ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="dataList">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="graphicsLabel">
        <property name="text">
         <string>イメージ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="CrossPattern">
          <color alpha="255">
           <red>200</red>
           <green>220</green>
           <blue>255</blue>
          </color>
         </brush>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="palLayout">
        <item>
         <widget class="QLabel" name="palLabel">
          <property name="text">
           <string>パレット</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="palList">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>グレースケール</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>リストに登録</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>ラベル</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="labelEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>アドレス</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="addrEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>パレット</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="palAddrEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>圧縮</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="compCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="regButton">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>登録</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="xTileBox">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="yTileBox">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ファイル</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dataList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>guiDataItemActivated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>guiRegButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>832</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>palList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>guiPalItemActivated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>guiDataItemActivated()</slot>
  <slot>guiRegButtonPressed()</slot>
  <slot>guiPalItemActivated()</slot>
 </slots>
</ui>
